tests/tests_create_lvmvdo_then_remove.yml

```diff
@@ -0,0 +1,70 @@
+---
+- hosts: all
+  become: true
+  vars:
+    storage_safe_mode: false
+    mount_location: '/opt/test1'
+    volume_group_size: '10g'
+    volume_size: '20g'
+    pool_size: '9g'
+
+  tasks:
+    - include_role:
+        name: linux-system-roles.storage
+
+    - include_tasks: get_unused_disk.yml
+      vars:
+        min_size: "{{ volume_group_size }}"
+        max_return: 1
+
+    - name: Create LVM VDO volume under volume group 'pool1'
```

Comment:
We need a test that passes `compression` and/or `deduplication` but not `vdo_pool_size` to make sure that works correctly.

Reply:
Fixed.
Extended create vdo test so it runs again without using `vdo_pool_size` option.